pkg/operator/ceph/object/rgw.go

```diff
@@ -75,7 +75,11 @@ func (c *clusterConfig) createOrUpdateStore(realmName, zoneGroupName, zoneName s
 	if err := c.startRGWPods(realmName, zoneGroupName, zoneName); err != nil {
 		return errors.Wrap(err, "failed to start rgw pods")
 	}
+	logger.Infof("created object store %q in namespace %q", c.store.Name, c.store.Namespace)
+	return nil
+}
 
+func (c *clusterConfig) enableRGWDashboard() error {
```

Comment:
`enableRGWDashboard` can be removed, in this case, `pkg/operator/ceph/object/controller.go` does not need to modified.
```
if !cephObjectStore.Spec.IsRGWDashboardEnabled() {
     return nil
}
```